browser/src/layer/tile/CalcTileLayer.js

```diff
@@ -258,6 +258,8 @@ L.CalcTileLayer = L.CanvasTileLayer.extend({
 
 		if (limitWidth || limitHeight || extendedLimit)
 			app.sectionContainer.requestReDraw();
+
+		this._syncTileContainerSize();
```

Comment:
shoudln't be done only if any change done?
do we redraw first (one line above) then resize again? is that ok?
didn't trigger two renderings?

Reply:
Thanks for the catch @eszkadev. I added a boolean variable to avoid duplicate calls to redraw.